docs/content/integrations/dagstermill/using-notebooks-with-dagster.mdx

```diff
@@ -1,9 +1,9 @@
 ---
-title: Using Notebooks with Jupyter, Papermill, and Dagster Tutorial
+title: Using Jupyter Notebooks with Papermill and Dagster Tutorial
 description: The Dagstermill package lets you run notebooks using the Dagster tools and integrate them into your data pipelines.
 ---
 
-# Using notebooks with Jupyter, Papermill, and Dagster
+# Using Jupyter Notebooks with Papermill and Dagster
```

Comment:
This just popped up in my notifications - sorry!

I don't think notebook is a proper noun, at least the guide doesn't use it as such. Headings and titles should use sentence case, so this should be `notebook`

Reply:
oops! on it